x-pack/plugins/license_management/public/components/telemetry_opt_in/telemetry_opt_in.js

```diff
@@ -107,7 +130,15 @@ export class TelemetryOptIn extends React.Component {
         {example}
         {toCurrentCustomers}
         <EuiCheckbox
-          label={<span>Send basic feature usage statistics to Elastic periodically. {popover}</span>}
+          label={
+            <span>
+              <FormattedMessage
+                id="xpack.licenseMgmt.telemetryOptIn.sendBasicFeatureStatisticsLabel"
+                defaultMessage="Send basic feature usage statistics to Elastic periodically."
```

Comment:
A space is missing after `periodically.`  This missing space makes the `Read more` link jam right up against the closing period.

Reply:
Thanks. I missed this space.